yt/utilities/lib/pixelization_routines.pyx

```diff
@@ -862,3 +862,107 @@ def pixelize_element_mesh(np.ndarray[np.float64_t, ndim=2] coords,
     free(vertices)
     free(field_vals)
     return img
+
+def element_mesh_line_plot(np.ndarray[np.float64_t, ndim=2] coords,
```

Comment:
Same deal here as above, should probably not say "plot" in the name since pixelizing a line could have other applications.